instrumentation/jodd-http-4.2/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/joddhttp/v4_2/JoddHttpHelper.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.joddhttp.v4_2;
+
+import static io.opentelemetry.javaagent.instrumentation.joddhttp.v4_2.JoddHttpSingletons.instrumenter;
+
+import io.opentelemetry.context.Context;
+import jodd.http.HttpRequest;
+import jodd.http.HttpResponse;
+
+public class JoddHttpHelper {
+
+  public static void doMethodExit(
+      Context context, HttpRequest request, Object result, Throwable throwable) {
```

Comment:
Is it possible for `result` to be of a different type than `HttpResponse`?
```suggestion
      Context context, HttpRequest request, HttpResponse result, Throwable throwable) {
```

Reply:
You are right, its always a `HttpResponse`. Since I now removed the `JoddHttpHelper` this is fixed already